### DISCUSSION TITLE
Merlin's Crystal - Lady of the Lake

### DISCUSSION THREAD

**Discussion by TheFatCat86** [runelite/runelite#15783] [Q&A] [answered]
I am doing the Merlin's Crystal quest and after speaking with the lady of the lake the quest does not move forward. I have gone backward on the quest to see if it would reset and unfortunately it is still not working. Any idea how to get it to move forward?

**Comment by kamielvf** [comment, marked as answer]
For issues with plugins from the plugin hub, such as the Quest Helper, please refer to their respective repository (https://github.com/Zoinkwiz/quest-helper). The link can also be found by clicking the ❔ icon in the plugin hub list.
